docs/conf/ssl.md

```diff
@@ -2,9 +2,121 @@
 
 SSL handling can be customized (or disabled) when creating a backend and is backend-specific.
 
-Depending on the underlying backend's client, you can customize SSL settings as follows:
+Depending on the underlying backend's client, you can customize SSL settings.
 
-* `HttpUrlConnectionBackend`: when creating the backend, specify the `customizeConnection: HttpURLConnection => Unit` parameter, and set the hostname verifier & SSL socket factory as required
-* akka-http: when creating the backend, specify the `customHttpsContext: Option[HttpsConnectionContext]` parameter. See [akka-http docs](http://doc.akka.io/docs/akka-http/current/scala/http/server-side/server-https-support.html)
-* async-http-client: create a custom client and use the `setSSLContext` method
-* OkHttp: create a custom client modifying the SSL settings as described [on the wiki](https://github.com/square/okhttp/wiki/HTTPS)
+## SSL Context
+
+Common requirement for handling SSL is creating `SSLContext`. It's required by several backends.
+
+Example assumes that you have your client key store in `.p12` format and the server certificate imported to your trust store.
```

Comment:
Looks great! Couple of questions:
* what is described here is importing certificates for a server, and hence implementing *one way SSL*, right? If so, it would be good to explicitly mention that - I suspect people might look for docs on "how to implement one way SSL" or "how to implement mutual SSL"
* sometimes you need to disable SSL - only during development of course ;) - I think a section on that would be helpful as well

Reply:
- It's actually a mutual SSL example since we have both client key store (`.p12` file) and trust store with server certificate imported. So server identifies client and vice versa. I can add an example without trust store (one way)
- I could add a code sample on how to disable SSL

Reply:
great :) I think it would be best to have sections on one-way / mutual / disabled SSL - I'm guessing that's what people will be looking for in the docs in the first place

Reply:
Done, but i didn't add explicit section about disabled SSL because that depends on server config as well.
On the client side we can disable validation of server certificates and that's included in "One way SSL" section.

Reply:
looks good, thank you!